cookbook/docs/contribute.rst

```diff
@@ -1,122 +1,321 @@
-      To register ``core`` examples, run the start command in the ``cookbook`` directory. So, ``cd`` to the required directory and run all the consequent commands in there!
+For example, to register
+`house price prediction <https://github.com/flyteorg/flytesnacks/tree/master/cookbook/case_studies/ml_training/house_price_prediction>`__
+example, run the start command in the ``ml_training`` directory. To register ``core`` examples, run the start command in
+the ``cookbook`` directory. So, ``cd`` to the required directory and run all the consequent commands in there!
 
-      Following are the commands to run if examples in ``core`` directory are to be tested on sandbox:
-        1. Build Docker container using the command: ``flytectl sandbox exec -- docker build . --tag "core:v1" -f core/Dockerfile``. 
-        2. Package the examples by running ``pyflyte --pkgs core package --image core:v1 -f``.
-        3. Register the examples by running ``flytectl register files --archive -p flytesnacks -d development --archive flyte-package.tgz --version v1``.
-        4. Visit https://localhost:30081/console to view the Flyte console, which consists of the examples present in the ``flytesnacks/cookbook/core`` directory.
-        5. To fetch new dependencies and rebuild the image, run 
-        ``flytectl sandbox exec -- docker build . --tag "core:v2" -f core/Dockerfile``, 
-        ``pyflyte --pkgs core package --image core:v2 -f``, and 
-        ``flytectl register files --archive -p flytesnacks -d development --archive flyte-package.tgz --version v2``.
-        6. Refer to `this doc <https://docs.flyte.org/projects/cookbook/en/latest/auto/larger_apps/larger_apps_iterate.html#quickly-re-deploy-your-application>`__ if the code in itself is updated and requirements.txt is the same.
+Testing ``core`` directory examples on sandbox
+-----------------------------------------------
+
+Build Docker container:
+
+.. prompt:: bash
+
+   flytectl sandbox exec -- docker build . --tag "core:v1" -f core/Dockerfile
+
+Package the examples by running
+
+.. prompt:: bash
+   
+   pyflyte --pkgs core package --image core:v1 -f
+
+Register the examples by running
+
+.. prompt:: bash
+   flytectl register files --archive -p flytesnacks -d development --archive flyte-package.tgz --version v1
+
+Visit ``https://localhost:30081/console`` to view the Flyte console, which consists of the examples present in the
+``flytesnacks/cookbook/core`` directory.
+
+To fetch new dependencies and rebuild the image, run 
+
+.. prompt:: bash
+
+   flytectl sandbox exec -- docker build . --tag "core:v2" -f core/Dockerfile
+   pyflyte --pkgs core package --image core:v2 -f
+   flytectl register files --archive -p flytesnacks -d development --archive flyte-package.tgz --version v2
+
+Refer `here <https://docs.flyte.org/projects/cookbook/en/latest/auto/larger_apps/larger_apps_iterate.html#quickly-re-deploy-your-application>`__
+if the code in itself is updated and requirements.txt is the same.
```

Comment:
Should we replace these commands with `pyflyte run ...`?

Reply:
sure, I'll think I'll leave that for someone else to re-write in another PR